ci-cd/p3_temp/test_ast_parser.sh

```diff
@@ -22,6 +22,26 @@ export PATH="${HOME}/.local/bin:${PATH}"
 pip install --user -r requirements.txt
 pip install --user -r requirements-dev.txt
 
+# Generate required polyglot_snippet_data.py files
```

Comment:
**N.B:** I thought about including precompiled versions of these, but I thought generating them would be a good way to (regularly) test the generation infrastructure itself.